module/VuFind/src/VuFind/Controller/MyResearchController.php

```diff
@@ -1725,6 +1725,23 @@ public function recoverAction()
         } elseif ($username = $this->params()->fromPost('username')) {
             $user = $userService->getUserByUsername($username);
         }
+        //ILS Driver:
+        //if the user hasn't logged in yet, but is found by the ILS, call function
+        //getPatronFromUsername
```

Comment:
Another minor style tweak that you can commit directly from this comment:

```suggestion
        // ILS Driver:
        // if the user hasn't logged in yet, but is found by the ILS, call function
        // getPatronFromUsername
```